source/declarations.tex

```diff
@@ -7006,8 +7006,10 @@
 \end{bnf}
 
 \pnum
-Every \grammarterm{namespace-definition} shall appear in the global scope
-or in a namespace scope\iref{basic.scope.namespace}.
+Every \grammarterm{namespace-definition} shall appear in namespace scope\iref{basic.scope.namespace}.
```

Comment:
Please say "at namespace scope" (which avoids the misunderstanding that someplace nested inside the namespace scope is fine, too (e.g. inside a class definition). Also, please drop the note that follows; in other situation, we never explicitly mention "global scope" when we say "at namespace scope".